snapcraft/internal/lifecycle.py

```diff
@@ -351,15 +351,18 @@ def snap(project_options, directory=None, output=None):
     # Check for our prerequesite external command early
     repo.check_for_command('mksquashfs')
```

Comment:
I think you can remove this now, no?

Reply:
It would be best to check really early in the game still for the default "run" case.

Reply:
I don't quite understand, but as long as you've considered it, that's good enough for me.